package.json

```diff
@@ -3,9 +3,14 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "-": "0.0.1",
```

Comment:
not sure what this is but pretty sure you don't need it

Reply:
removed